Code_Integrafion_XSOAR.py

```diff
@@ -141,6 +144,8 @@ def main():
             return_results(pfsense.get_aliases(alias_id))
         elif command == 'pfsense-create-alias':
             data_alias = pfsense.input_data(args, is_rule=False)
+            data_alias['address'] = pfsense.split_data(data_alias['address'])

```

Comment:
for key in ['address', 'detail']:
    data_alias[key] = pfsense.split_data(data_alias[key])
    
Tối ưu lại code